kv/dist_sender_server_test.go

```diff
@@ -311,7 +311,7 @@ func TestReverseScanWithSplitAndMerge(t *testing.T) {
 
 	// Case 2: encounter with range merge .
 	// Merge the range ["e", "g") and ["g", "\xff\xff") .
-	if err := db.AdminMerge("g"); err != nil {
+	if err := db.AdminMerge("e"); err != nil {
```

Comment:
AdminMerge takes any key in the left-hand side of the range, but the split key is in the right-hand side. This call was trying to merge `["g", "\xff\xff"]` with its non-existent following range instead of merging the two ranges mentioned in the comment. Now that `AdminMerge` returns an error in this case instead of silently doing nothing, it detected this bug.